PMM-8259 Update custom message for server error

floatinginbits opened the pull request:
In case of 401 or 403 HTTP status response from server, pmm-admin
appends a custom message to error response from server. The message
causes mismatch between HTTP status code and updated error message.

With this change, we will provide custom message only for 401 HTTP status code. This will avoid mismatch between HTTP status code and additional error message.

Commit:
PMM-8259 Remove custom message for server error

In case of 401 or 403 HTTP status response from server, pmm-admin
appends a custom message to error response from server. The message
causes mismatch between HTTP status code and updated error message.
We expect server to provide a descriptive error message, so appended
message is not required. This patch removes the custom error message.

it-percona-cla commented:
[![CLA assistant check](https://cla-assistant.percona.com:443/pull/badge/signed)](https://cla-assistant.percona.com:443/percona/pmm-admin?pullRequest=166) <br/>All committers have signed the CLA.

Commit:
Update Travis configuration due to AWS ECR limit

AWS ECR limits number of requests without authorization. Travis
currently doesn't have authorization. Hence we fail to get container
image for Travis build, leading to failure in CI pipeline. This
patch updates container source by reverting changes made by PMM-8309.

BupycHuk commented on main.go:
```diff
@@ -189,9 +189,6 @@ func main() {
 			msg := e.Error
-			if e.Code == 401 || e.Code == 403 {
-				msg += ". Please check username and password."
-			}
```
we should keep this suffix for 401.

floatinginbits replied:
Is there an advantage for retaining a check for 401 HTTP status code? We can keep a check here or ensure that error messages from source provides enough information to take corrective action.

BupycHuk replied:
Yes, 401 means unauthorized, so it makes sense.

floatinginbits replied:
Done

Commit:
PMM-8259 Add custom message for 401 from server

Whenever server returns 401 HTTP status code, a custom error message
is appended to server response.
